docs/source/conf.py

```diff
@@ -37,7 +37,8 @@
 extensions = [
     "sphinx_rtd_theme",
     "sphinx.ext.autodoc",
-    "sphinx.ext.napoleon",
+    "sphinx.ext.napoleon",  # https://github.com/tox-dev/sphinx-autodoc-typehints/issues/15
+    "sphinx_autodoc_typehints",  # Must import napoleon before typehints
```

Comment:
Is napoleon just needed as a dependency of typehints, or also because you use some different style of docstrings than the existing codebase?

Reply:
[You yourself](https://github.com/PyFstat/PyFstat/blame/master/docs/source/conf.py#L40) introduced Napoleon, and I belive it's because of the way we are formatting the docstrings.

Reply:
I misread the diff, thought you were adding it.